ui/craco.config.js

```diff
@@ -3,56 +3,56 @@ const { ModuleFederationPlugin } = require("webpack").container;
 const { "react-lazylog": undefined, ...sharedDeps } = require("./package.json").dependencies;
 
 module.exports = ({ env }) => ({
```

Comment:
For some reason the original file was overindented with too many spaces. My IDE just corrected the indentation.